File: core/src/main/java/forpleuvoir/dhwu1a/core/common/IJsonData.java
package forpleuvoir.dhwu1a.core.common;

import forpleuvoir.dhwu1a.core.util.JsonUtil;

/**
 * @author forpleuvoir
 * <p>#project_name dhwu1a
 * <p>#package forpleuvoir.dhwu1a.core.common
 * <p>#class_name IJsonData
 * <p>#create_time 2021/6/30 0:38
 */
public interface IJsonData {

    default String toJsonString() {
        return JsonUtil.toJsonStr(this);
    }
}

File: core/src/main/kotlin/forpleuvoir/dhwu1a/core/common/IJsonData.kt
package forpleuvoir.dhwu1a.core.common

import forpleuvoir.dhwu1a.core.util.toJsonStr

/**
 * @author forpleuvoir
 *
 * #project_name dhwu1a
 *
 * #package forpleuvoir.dhwu1a.core.common
 *
 * #class_name IJsonData
 *
 * #create_time 2021/6/30 0:38
 */
interface IJsonData {

      fun toJsonString(): String {
        return this.toJsonStr()
    }
}